hr_period/views/hr_fiscalyear_view.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Fiscal Year -->
+    <record id="view_hr_fiscalyear_form" model="ir.ui.view">
+        <field name="name">hr.fiscalyear.form</field>
+        <field name="model">hr.fiscalyear</field>
+        <field name="arch" type="xml">
+            <form string="Fiscal Year">
+                <header>
+                    <button name="create_periods" string="Create Periods" type="object" class="oe_highlight" states="draft"/>
```

Comment:
`oe_highlight` no longer has CSS rules associated with it, it is being replaced by `btn-primary` since version 12.0 [[source](https://github.com/odoo/odoo/blob/1e5fbb8e5bf0e0458d83a399b2b59d03a601e86a/addons/web/static/src/js/core/dom.js#L340-L345)].

Same problem in:

- below in this file;
- `views/hr_payslip_run_view.xml`.